### PR TITLE
Added populations served filter to search results 

### PR DESCRIPTION
**Pull request by SMakaiTakori**
## Changes

Added populations served filter to search results for desktop and mobile. 

## Screenshots

[OwnPath populations served filter video](https://www.loom.com/share/2cd33301c8c040e1a61f9f85f785e85e)

## Checklist

- [x] if changes filter/search functionality, adds or updates unit tests
- [x] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)
- ~[ ] if adds a new page, adds accessibility tests~
- ~[ ] if adds a new page or new interaction, adds e2e test~
- ~[ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)~

Fixes [Add "Populations served" filter](https://app.asana.com/0/1202105544020589/1203960859378993/f)

### CODE REVIEW

**Review by alex-cds — COMMENTED** [review]
@SMakaiTakori Nice work on this!! So speedy. 

Is it possible to remove the ages from this? Feels duplicative since we already have an Age filter. 

I'm also interested in changing the order of the filters. Right now they aren't alphabetical. I'd love to try and match a hypothetical mental model of what someone might be thinking while they search. Let me know your preference for this or a new ticket! :)

**Comment by SMakaiTakori** [comment]
> @SMakaiTakori Nice work on this!! So speedy.
> 
> Is it possible to remove the ages from this? Feels duplicative since we already have an Age filter.
> 
> I'm also interested in changing the order of the filters. Right now they aren't alphabetical. I'd love to try and match a hypothetical mental model of what someone might be thinking while they search. Let me know your preference for this or a new ticket! :)

@alex-cds that makes sense! I will go ahead and remove the older adults, minors, and youth options from this filter. I can alphabetize the filters while I am doing my code cleanup. Thanks for the review!

File: src/components/Search/Filters/PopulationsServedInput.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  options,
+  filters,
+  setFilters,
+}: PopulationsServedInputProps) {
+  const { t } = useTranslation();
+
+  const setPopulationServedFilter = (populationServed: PopulationsServed) => {
+    logEvent(AnalyticsAction.UpdateFilter, {
+      label: "populations served",
+      filter_type: "populations served",
```
Comment (jlhogan) [inline src/components/Search/Filters/PopulationsServedInput.tsx:29]: will need to create this filter value in GA, I can do that

**Review by jlhogan — APPROVED** [review]
LGTM! filters are indeed supposed to be in alphabetical order (did I mess that up? whoops!) 
Maybe add a comment at the top of the desktop + mobile filter declarations to note that new filters added should retain alphabetical ordering? (in `MobileControl.tsx` and `DesktopControlToggles.tsx`)

**Comment by SMakaiTakori** [comment]
@alex-cds 
I went ahead and took out the age filters from desktop and mobile in the populations served filter PR that's up right now. I decided to leave the order of the filters as they are since the team is still deciding on the mental model that's best. I figure a separate ticket can be made for this since Julia has an idea already about automating the alphabetical order or however we end up doing it. When you get a chance, please take a look at the current deploy preview and let me know if it looks good to merge in from your UX perspective. Thanks!

**Review by alex-cds — APPROVED** [review]
Thank you @SMakaiTakori ! Have a great weekend.

**Pull request merged**